### ISSUE TITLE
rmvpe+

### ISSUE THREAD

**Issue by Neovolter** [SociallyIneptWeeb/AICoverGen#109]
I've found in RVC studio modified algorithm rmvpe+. It would be great to have it in AICoverGen too)

**Comment by smartinezbragado** [comment]
can you share the link?

**Comment by JackismyShephard** [comment]
> I've found in RVC studio modified algorithm rmvpe+. It would be great to have it in AICoverGen too)

I am going to add add this to my fork of this repo (https://github.com/JackismyShephard/ultimate-rvc) in the near future

**Comment by Bebra777228** [comment]
> > I've found in RVC studio modified algorithm rmvpe+. It would be great to have it in AICoverGen too)
> 
> I am going to add add this to my fork of this repo (https://github.com/JackismyShephard/ultimate-rvc) in the near future

Yes, I'm also planning to add **rmvpe+** to my fork shortly (it might even be tomorrow). You can find the fork at this link: [**Pol-Litres-RVC/CoverGen**](https://github.com/Bebra777228/Pol-Litres-RVC/tree/CoverGen). **_Please note that the fork is written in Russian and primarily intended for Russian-speaking users_**. Feel free to have a look if you're interested =)
